pkg/gadgets/trace/capabilities/tracer/tracer_test.go

```diff
@@ -218,6 +218,48 @@ func TestCapabilitiesTracer(t *testing.T) {
 				}
 			},
 		},
+		"unique_false": {
+			getTracerConfig: func(info *utilstest.RunnerInfo) *tracer.Config {
+				return &tracer.Config{
+					MountnsMap: utilstest.CreateMntNsFilterMap(t, info.MountNsID),
+					Unique:     false,
+				}
+			},
+			generateEvent: repeatChown,
+			validateEvent: func(t *testing.T, info *utilstest.RunnerInfo, _ interface{}, events []types.Event) {
+				nfound := 0
+				for _, event := range events {
+					if event.CapName == "CAP_CHOWN" {
+						nfound++
+					}
+				}
+
+				if nfound <= 1 {
+					t.Fatalf("Not enough events found ")
+				}
+			},
+		},
+		"unique_true": {
+			getTracerConfig: func(info *utilstest.RunnerInfo) *tracer.Config {
+				return &tracer.Config{
+					MountnsMap: utilstest.CreateMntNsFilterMap(t, info.MountNsID),
+					Unique:     true,
+				}
+			},
+			generateEvent: repeatChown,
+			validateEvent: func(t *testing.T, info *utilstest.RunnerInfo, _ interface{}, events []types.Event) {
+				nfound := 0
+				for _, event := range events {
+					if event.CapName == "CAP_CHOWN" {
```

Comment:
If we merge https://github.com/kinvolk/inspektor-gadget/pull/1035, it should be changed to "CHOWN".